### PR TITLE
[Update] webui-popover

### PR DESCRIPTION
**Pull request by MattIPv4**
Fix to an existing library.

The library makes reference to the img folder from assets inside the dist folder so it will be best for us to include both the dist & img folders.

### CODE REVIEW

**Comment by MattIPv4** [comment]
@PeterDaveHello What is the best way to fix this? All versions we have are missing the required img folder, but a restructure would mean all existing usage breaks?

File: ajax/libs/webui-popover/package.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-  "filename": "jquery.webui-popover.min.js",
+  "filename": "dist/jquery.webui-popover.min.js",
```
Comment (PeterDaveHello) [inline ajax/libs/webui-popover/package.json]: Should not touch this line

**Comment by PeterDaveHello** [comment]
I think we should let the upstream fix it as it's beyond our topics?

**Comment by MattIPv4** [comment]
@PeterDaveHello This isn't really an issue with upstream though, we just aren't using the correct structure from the upstream?

**Comment by PeterDaveHello** [comment]
@MattIPv4 I mean in convention or common cases, dist is for distribution, reference a file not being distributed doesn't make sense.

**Comment by MattIPv4** [comment]
@PeterDaveHello Yeah, fair enough, I was just going for the approach of resolving this ourselves instead of waiting on upstream to resolve it.

Should we just an open an issue with the upstream provider instead?

**Comment by PeterDaveHello** [comment]
@MattIPv4 yes I prefer that, and in most cases, upstream would also like to know the issues, unless reaching out doesn't work at all, I prefer not to fix or add workaround by our side, thanks.

**Pull request closed without merging**